### PR TITLE
FAQ - some tweaks

### PR DESCRIPTION
**Pull request by twMat**

### CODE REVIEW

File: wiki/tiddlers/FAQ.tid [hunk 1]
```diff
@@ -2,47 +2,46 @@ title: FAQ
...
 !! I love this! Can I use for production/very important things?
-I would love to answer yes, but for now please don't do it. Noteself is still on Beta stage, and has to be extensively tested.
-But, if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give it a try.
+We would love to answer yes, but for now please don't do it. Noteself is still in Beta stage and has to be extensively tested.
+But if you are an early adopter and want to try new and exciting technologies NoteSelf is stable enough to give a try.
 
 !!Where is my data stored?
-Your data is stored inside your browser. We use PouchDB in the background,
-which creates a database on your browser's LocalStorage.
+Your data is stored inside your browser. We use PouchDB in the background, which creates a database in your browser's LocalStorage.
```
Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: I'm curious about why you have joined this into one line. Tiddlywiki treats line breaks as spaces, which was my intention

File: wiki/tiddlers/FAQ.tid [hunk 2]
```diff
@@ -2,47 +2,46 @@ title: FAQ
...
+!! How can I install regular TiddlyWiki plugins?
+It is easy! Just go to the `Control Panel`, open the plugins tab and open the plugin library.
+After a plugin is installed all you have to do is refresh the page. No need to download the file or save the wiki.
 
 If you want it //"hardcoded"// into your file, install the plugin and ''after that'' open the `Control Panel` ( ,,{{$:/core/images/options-button}},, ),
  navigate to the `TiddlyPouch` tab and click on `Save config`.
 
 !! I'm a regular TW user. What are the advantages of NoteSelf?
-* Multiple devices synchronization
+* Multiple device synchronization
```
Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: Don't you think it is better "Multi device synchronization" ?

File: wiki/tiddlers/FAQ.tid (hunk 1 above)
Comment (twMat) [inline wiki/tiddlers/FAQ.tid]: Probably just a mistake, sorry.

File: wiki/tiddlers/FAQ.tid (hunk 2 above)
Comment (twMat) [inline wiki/tiddlers/FAQ.tid]: Hm, actually I think it should say "Synchronization between multiple devices" or "Synchronize  multiple devices".
("Multi device" sounds like "multi-tool" which is just *one* thing that does many things.)

Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: I agree, would you mind to reformulate it to your first suggestion? "Synchronization between multiple devices"

File: wiki/tiddlers/FAQ.tid (hunk 1 above)
Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: Would you mind to split them again so I can accept the pull request?

Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: Oops, already merged :P forget about what I said

File: wiki/tiddlers/FAQ.tid (hunk 2 above)
Comment (twMat) [inline wiki/tiddlers/FAQ.tid]: I'm afraid I don't know how to do that. Some brief attempts failed.

Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: Do you mean to re-write the sentence or to make another pull request ? This one is closed, you should open another one, which I didn't said clearly . Maybe is that what you mean ?

Comment (twMat) [inline wiki/tiddlers/FAQ.tid]: Ok, so it's not possible to edit closed PRs. That's what I tried.
Ok I posted a new PR. But, isn't it a lot easier for you to do such a tweak than it is for me? I had to once again locate where the FAQ tid was stored, and go through the rewrite in my own repo and then post it to you, whereas you probably could change it in a snap, no?

Comment (danielo515) [inline wiki/tiddlers/FAQ.tid]: To edit a single file you don't need to go to your fork. You can locate the file on my repo, click the edit button and GitHub will create a new branch automatically for you and allow you to make a pull request directly from that branch. Isn't that the flow you are following?
